crates/router/src/connector/shift4.rs

```diff
@@ -77,7 +77,7 @@ impl ConnectorCommon for Shift4 {
             .change_context(errors::ConnectorError::FailedToObtainAuthType)?;
         Ok(vec![(
             headers::AUTHORIZATION.to_string(),
-            auth.api_key.into_masked(),
+            format!("Basic {}", auth.api_key).into_masked(),
```

Comment:
will this break existing api keys that are already stored in DB?
Are we planing a migration for existing shift4 connectoraccounts?
or maybe add a backwards compatiblity here s.t
```
if api_key.startswith("Basic ") {
    api_key.into_masked()
} else {
    format!("Basic {}", auth.api_key).into_masked(),
}

Reply:
Yes, we want to have a migration for this. Till then, I'll be marking this `S-blocked`.

Reply:
@SanchithHegde should i make this backwards compatible then?

Reply:
@nishanthanr151998 No worries, we'll take it from here. We'll anyway have to do a similar migration for the other connectors.

In the meantime, feel free to pick up other issues. Thanks for your PR!